en/technical-reference/going-live.md

```diff
@@ -3,30 +3,37 @@
 
 This document is designed to assist you in switching from *Sandbox* environment to *Production* environment, in order to start accepting real world payment for your business. This is also known as "Going Live".
 
+?>Make sure you have already completed the test payment using our [Payment Simulator](/en/technical-reference/sandbox-test.md) in the Sandbox environment to ensure that your implementation is working fine.
 
-## 1. Login to Your Midtrans Production Environment Dashboard
+## 1. Checking Your Production Account
+To make sure your payment channels on Production account is already active, follow the steps given below.
 
-To login to your Midtrans *Production* environment, follow the steps given below.
+1. Login to your Midtrans Administrative Portal using *Production* environment at https://dashboard.midtrans.com/login.
 
-1. Login to your account at https://account.midtrans.com. 
-    - refer to [Access Midtrans Administration Portal](/en/midtrans-account/overview.md#accessing-midtrans-administration-portal) for more details.
-2. Select **Production** Environment from top-left environment switcher dropdown.
-    - refer to [Switching Environment](/en/midtrans-account/overview.md#switching-environment) for more details.
+2. To make sure your payment channels on Production account is active. 
+    - Go to menu **PAYMENT METHODS** from the dashboard page
+    - *Active Payment Methods* page is displayed.
+    - Then you can find a list of active payment methods ready in a production environment
 
-## 2. Checking Your Production Account
+![Active Payment Method](./../../asset/image/active-payment-methods.png)
+<br>
+
+?>refer to [Access Midtrans Administration Portal](/en/midtrans-account/overview.md#accessing-midtrans-administration-portal) for more details.
+
+## 2. Retrieving Production Server Key and Client Key
+To switch from Sandbox to Production environment, Client Key and Server Key production are required.
+
+To get access to the Client Key and Server Key, follow the steps given below.
 
-To make sure your production account is active, follow the steps given below.
+1. Login to your MAP account.
 
```

Comment:
I personally prefer to avoid using `MAP` terms as not all merchants maybe familiar with the terms, and would use `your Midtrans Account` as it is more straight forward to understand.